tools/cbmc/patches/compute_patch.py

```diff
@@ -93,11 +93,18 @@ def manipulate_headerfile(defines, header_file):
             if (match and
                     match.group(1) in defines and
                     not last.lstrip().startswith("#ifndef")):
+                full_def = [line]
```

Comment:
Why don't you use directly string concatenation instead of collecting lines in a list and joining it with an empty string?
```
full_def = line
while line.rstrip().endswith("\\"):
    full_def += next(source)
```

Reply:
I have updated the code with your suggestion, thanks!